.travis.yml

```diff
@@ -13,6 +13,9 @@ script:
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
   - julia -e 'Pkg.clone(pwd()); Pkg.checkout("DataFrames", "dfk/statsmodel-purge"); Pkg.build("StatsModels"); Pkg.test("StatsModels"; coverage=true)'
 after_success:
+  # build and deploy documentation with Documenter.jl
+  - openssl aes-256-cbc -K $encrypted_a9d536040c76_key -iv $encrypted_a9d536040c76_iv -in docs/.documenter.enc -out docs/.documenter -d
```

Comment:
Also, in case it's important, I have v1.8.4 (the latest release) of the travis cli, installed via

    $ gem install travis -v 1.8.4 --no-rdoc --no-ri

(as per the [instructions on their github page](https://github.com/travis-ci/travis.rb#installation))